templates/comments/list.html

```diff
@@ -94,9 +94,19 @@ <h2><i style="padding-right: 0.3em" class="fa fa-comments"></i>{{ _('Comments')
                             <div class="content content-description">
                                 <div class="comment-title"{% if not node.title or node.title.startswith('Re:') %}
                                      style="display: none"{% endif %}>{{ node.title }}</div>
-                                <div class="comment-body">
-                                    {{ node.body|markdown('comment', MATH_ENGINE, True)|reference|str|safe }}
-                                </div>
+                                    <div class="comment-body" {% if node.score <= vote_hide_threshold %}style="display:none"{% endif %}>
+                                        {{ node.body|markdown('comment', MATH_ENGINE, True)|reference|str|safe }}
+                                    </div>
+                                    {% if node.score <= vote_hide_threshold %}
```

Comment:
I thought this was supposed to go first? Or does it not actually matter?

Reply:
Order doesn't matter there, but I messed up the order of https://github.com/DMOJ/site/blob/hiding_comments/templates/comments/media-js.html#L200 and https://github.com/DMOJ/site/blob/hiding_comments/templates/comments/media-js.html#L201